src/webrtc/call.ts

```diff
@@ -1404,7 +1436,7 @@ export class MatrixCall extends TypedEventEmitter<CallEvent, CallEventHandlerMap
             return this.isMicrophoneMuted();
         }
 
-        if (!this.hasLocalUserMediaAudioTrack && !muted) {
+        if (!this.hasUserMediaAudioSender && !muted) {
```

Comment:
The fact we have an audio sender doesn't mean we have a track to enable, right?

Reply:
True, we should probably check both